examples/hybrid_jobs/6_QNSPSA_optimizer_with_embedded_simulator/source_scripts/QNSPSA.py

```diff
@@ -0,0 +1,227 @@
+import random
+import pennylane as qml
+from pennylane import numpy as np
+from scipy.linalg import sqrtm
+
+
+class QNSPSA:
+    """Quantum natural SPSA optimizer. Refer to https://arxiv.org/abs/2103.09232
```

Comment:
This has been published and you should replace with citation: https://quantum-journal.org/papers/q-2021-10-20-567/

Reply:
It seems this link (along with some other ones) from Quantum is failing. For now, I would suggest using the arxiv one.

Reply:
I mean use the actual citation, possibly with a link.